packages/vue-cli-plugin-vuetify/util/helpers.js

```diff
@@ -32,6 +32,19 @@ function fileExists (api, file) {
   return fs.existsSync(api.resolve(file))
 }
 
+// Returns the correct path for styles based on the Vuetify version
+function determineStylesPath(end) {
```

Comment:
might be splitting hairs, but what about `getStylesPath`? It's the typical language that we use in Vuetify.

Reply:
Let's do it, I was having trouble coming up with a name tbh lol